src/java/picard/analysis/artifacts/ConvertSequencingArtifactToOxoG.java

```diff
@@ -20,12 +20,42 @@
 import java.util.Set;
 
 @CommandLineProgramProperties(
-        usage = ConvertSequencingArtifactToOxoG.USAGE,
-        usageShort = ConvertSequencingArtifactToOxoG.USAGE,
+        usage = "Converts all sequencing artifact errors including all bait-bias and pre-adapter artifacts to " +
+                "8-oxoguanine artifacts <br /><br />." +
+                "Oxidation of guanine to 8-oxoguanine can occur during library preparation of genomic DNA via a " +
+                "combination of heat, shearing, and metal contaminates in a sample (doi: 10.1093/nar/gks1443).  It is " +
+                "one of the most common artifacts associated with genomic library preparation." +
+                "In DNA, 8-oxoguanine can pair with " +
+                "either cytosine or adenine, ultimately leading to G→T transversion mutations during PCR amplification.  " +
+                "The variants (C→A)/(G→T) tend to occur in specific sequence contexts e.g. CCG→CAG" +
+                "(doi:10.1093/nar/gks1443).  Although occurring at relatively low frequencies," +
+                " these artifacts can have profound impacts on variant calling fidelity (doi:10.1093/nar/gks1443).  " +
+                "<br /><br />This tool calculates the Phred-scaled probability that an alternate base" +
+                " call results from an oxidation artifact.  This probability score is based on base context, sequencing" +
+                " read orientation, and the characteristic low allelic frequency (doi:10.1093/nar/gks1443).  Lower " +
+                "probability values implicate artifacts resulting from 8-oxoguanine, while higher probability values" +
+                " suggest that an alternate base call is due to either some other type of artifact or is a real variant." +
+                "<br /><br />" +
+                "Output from CollectSequencingArtifactMetrics is the input for this tool.  Only the prefix or base" +
```

Comment:
The explanation of how file names are generated from a basename is a little convoluted, could be simplified.